core/cmd/eth_keys_commands_test.go

```diff
@@ -320,6 +321,7 @@ func TestClient_ImportExportETHKey_NoChains(t *testing.T) {
 	set = flag.NewFlagSet("test Eth export invalid id", 0)
 	set.Parse([]string{"999"})
 	set.String("newpassword", "../internal/fixtures/apicredentials", "")
+	set.Bool("bypass-version-check", true, "")
```

Comment:
I think that these depended on running _after_ the racy tests had mutated the global static values. They started failing consistently after resolving the race.

Reply:
Very interesting! Does this mean that the different test files are not run independently and share the static values?

Reply:
Yeah. Packages are tested as a whole in go. Also:

> Test files that declare a package with the suffix "_test" will be compiled as a separate package, and then linked and run with the main test binary.

https://pkg.go.dev/cmd/go#hdr-Test_packages